internal/pkg/describe/webapp.go

```diff
@@ -22,6 +22,14 @@ type WebAppURI struct {
 	Path    string // Empty if the application is served on HTTPS. Otherwise, the pattern used to match the application.
 }
 
+// WebAppDeployInfo contains ECS deploy information of a web application.
+type WebAppDeployInfo struct {
```

Comment:
🤔 Can we think of more specific terms? "Info" feels too vague.

How about something like this? that mentions ECS and web app:
```go
type WebAppECSParams struct { 
   taskSize
   ContainerPort string
   TaskCount int
}

type taskSize struct {
   CPU string
   Memory string
}
```

Reply:
Addressed!